src/prefect/client/schemas/actions.py

```diff
@@ -66,6 +67,11 @@ class FlowCreate(ActionBaseModel):
         description="A list of flow tags",
         examples=[["tag-1", "tag-2"]],
     )
+    labels: Union[KeyValueLabels, None] = Field(
+        default_factory=dict,
+        description="A dictionary of key-value labels. Values can be strings, numbers, or booleans.",
+        examples=[{"key": "value1", "key2": 42}],
+    )
```

Comment:
Could these models use a `KeyValueLabelsField` too?

Reply:
Yeah, that's good idea. I'll see where all I can use that Field class.